### PR TITLE
feat(cross-validation): Add `fit_time_per_data_point` and `score_time_per_data_point` metrics

### PR DESCRIPTION
**Pull request by augustebaum**
Closes https://github.com/probabl-ai/skore/issues/858

### CODE REVIEW

File: skore/src/skore/sklearn/cross_validation/plot.py [hunk 1]
```diff
@@ -56,7 +60,7 @@ def plot_cross_validation(cv_results: dict) -> plotly.graph_objects.Figure:
         bar_color = plotly.colors.qualitative.Plotly[
             col_i % len(plotly.colors.qualitative.Plotly)
         ]
-        bar_x = [min(df.index) - 0.5, max(df.index) + 0.5]
+        bar_x = numpy.linspace(min(df.index) - 0.5, max(df.index) + 0.5, num=10)
```
Comment (thomass-dev) [inline skore/src/skore/sklearn/cross_validation/plot.py:63]: Do you think we can do the same using only stdlib?

Comment (augustebaum) [reply skore/src/skore/sklearn/cross_validation/plot.py:63]: sure -- numpy is in our transitive dependencies (because of sklearn) but I can rework it

Comment (augustebaum) [reply skore/src/skore/sklearn/cross_validation/plot.py]: done

Comment (thomass-dev) [reply skore/src/skore/sklearn/cross_validation/plot.py]: Nice, thanks!

**Review by thomass-dev — APPROVED** [review]

**Review by thomass-dev — APPROVED** [review]

**Pull request merged**